zebrad/tests/acceptance.rs

```diff
@@ -1212,6 +1212,51 @@ async fn rpc_endpoint() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn non_blocking_logger() -> Result<()> {
+    let _init_guard = zebra_test::init();
+
+    // Write a configuration that has RPC listen_addr set
+    // [Note on port conflict](#Note on port conflict)
+    let mut config = random_known_rpc_port_config()?;
```

Comment:
We might need to set `config.tracing.filter` to "debug" or "trace" to get enough log output in this test.